[agent/logs/docker] Document the docker tailing from file feature (#9789)
DataDog/documentation: content/en/integrations/faq/compose-and-the-datadog-agent.md

      port: 6379
```

> The `Dockerfile` is used to instruct Docker compose to build a Datadog Agent image including the `redisdb.yaml` file at the right locatiom:

Reviewer: ```suggestion
The `Dockerfile` is used to instruct Docker compose to build a Datadog Agent image including the `redisdb.yaml` file at the right location:
```

```
FROM gcr.io/datadoghq/agent:latest
ADD conf.d/redisdb.yaml /etc/datadog-agent/conf.d/redisdb.yaml
```

### Log collection

> The `docker-compose.yml` can be extended to allow the Datadog agent to collect container logs.

Reviewer: ```suggestion
The `docker-compose.yml` can be extended to allow the Datadog Agent to collect container logs.
```

```yaml
version: '3'
services:
  redis:
    image: redis
    labels:
      com.datadoghq.ad.logs: '[{"source": "redis", "service": "redis"}]'
  datadog:
    build: datadog
    links:
     - redis # Connect the Datadog Agent container to the Redis container
    environment:
     - DD_API_KEY=${DD_API_KEY}
     - DD_SITE={{< region-param key="dd_site" >}}
     - DD_LOGS_ENABLED=true
     - DD_LOGS_CONFIG_DOCKER_CONTAINER_USE_FILE=true # To read log from file instead of the docker socket
    volumes:
     - /var/run/docker.sock:/var/run/docker.sock
     - /proc/:/host/proc/:ro
     - /sys/fs/cgroup:/host/sys/fs/cgroup:ro
     - /var/lib/docker/containers:/var/lib/docker/containers:ro
```

> **Note**: This only collect logs from the `Redis` container, logs can be collected from the Datadog agent by addign a similar `com.datadoghq.ad.logs` label. Log collection can be also explicitely enabled for all container by setting the environment variable `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` to `true`. See the complete [docker log collection documentation][4] for additional details.

Reviewer: ```suggestion
**Note**: This only collect logs from the `Redis` container. Logs can be collected from the Datadog Agent by adding a similar `com.datadoghq.ad.logs` label. Log collection can be also explicitly enabled for all containers by setting the environment variable `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` to `true`. See the complete [Docker log collection documentation][4] for additional details.
```
